libkbfs/folder_branch_ops.go

```diff
@@ -2241,6 +2247,31 @@ func (fbo *folderBranchOps) doMDWriteWithRetry(ctx context.Context,
 				if err = fbo.cr.Wait(ctx); err != nil {
 					return err
 				}
+			} else if isRevisionConflict(err) {
```

Comment:
I think it has to be retriable, because all of the block IDs will be different when there's a different predecessor, and we have to reconstruct the entire MD when we set the new head.  Right?